### PR TITLE
Update OAP e2e cases and images

### PR DESCRIPTION
**Pull request by lujiajing1126**
This PR updates e2e cases with SW OAP

### CODE REVIEW

**Comment by hanahmily** [comment]
LGTM. @kezhenxu94 Would you please take a look?

File: test/e2e-v2/script/env [hunk 1]
```diff
@@ -13,17 +13,18 @@
...
 SW_AGENT_CLIENT_JS_TEST_COMMIT=4f1eb1dcdbde3ec4a38534bf01dded4ab5d2f016
 SW_KUBERNETES_COMMIT_SHA=0f3ec68e5a7e1608cec8688716b848ed15e971e5
-SW_ROVER_COMMIT=90c93c706743aac1f5853b677730edae8cc32a2c
-SW_CTL_COMMIT=219876daf985fd474955834ef0b65013f0890e96
+SW_ROVER_COMMIT=d956eaede57b62108b78bca48045bd09ba88e653
 
-SW_OAP_COMMIT=dc39ce9bb44ed33d9c2bb0d5a054b1dfd5bbd657
+SW_CTL_COMMIT=e684fae0107045fc23799146d62f04cb68bd5a3b
+
+SW_OAP_COMMIT=109e6dea1672647992f693c5755b376fbbab048b
```
Comment (wu-sheng) [inline test/e2e-v2/script/env:29]:
```suggestion
SW_OAP_COMMIT=d5388683322ee6a4aed2a3bc29d439aadfca9a04
```

At least should be a released version in the integration.

File: test/e2e-v2/script/env [hunk 2]
```diff
@@ -13,17 +13,18 @@
...
 SW_AGENT_CLIENT_JS_COMMIT=af0565a67d382b683c1dbd94c379b7080db61449
 SW_AGENT_CLIENT_JS_TEST_COMMIT=4f1eb1dcdbde3ec4a38534bf01dded4ab5d2f016
 SW_KUBERNETES_COMMIT_SHA=0f3ec68e5a7e1608cec8688716b848ed15e971e5
-SW_ROVER_COMMIT=90c93c706743aac1f5853b677730edae8cc32a2c
-SW_CTL_COMMIT=219876daf985fd474955834ef0b65013f0890e96
+SW_ROVER_COMMIT=d956eaede57b62108b78bca48045bd09ba88e653
 
-SW_OAP_COMMIT=dc39ce9bb44ed33d9c2bb0d5a054b1dfd5bbd657
+SW_CTL_COMMIT=e684fae0107045fc23799146d62f04cb68bd5a3b
+
```
Comment (wu-sheng) [inline test/e2e-v2/script/env:28]:
```suggestion
```

Is this empty line on purpose?

Comment (lujiajing1126) [reply test/e2e-v2/script/env]: Removed

**Review by wu-sheng — APPROVED** [review]

**Review by kezhenxu94 — APPROVED** [review]

**Pull request merged**